packs/feat-effects/effect-offensive-boost.json

```diff
@@ -35,10 +35,27 @@
                 ],
                 "prompt": "PF2E.SpecificRule.Prompt.Weapon"
             },
+            {
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Inventor.PersistentBoost.ConstructLabel",
+                "option": "persistent-boost",
+                "predicate": [
+                    "self:trait:minion",
+                    {
+                        "gte": [
+                            "self:level",
+                            16
+                        ]
+                    }
+                ],
+                "priority": 51,
+                "toggleable": true
+            },
```

Comment:
I think we can predicate on the feat being in the item's parent roll options maybe?

Reply:
I'll give that a go

Reply:
I ended up adding a roll option to Persistent Boost so it could be used by the effect. Cleaner than the toggle, at least.

Reply:
I had an idea, so I came back to this one. Instead of using a `self:` roll option to denote that the Inventor took Persistent Boost, an Item Alteration can add a tag to the Persistent Boost feature, and that's something the persistent damage on the effect can be predicated on.